bolt-connection/test/bolt/bolt-protocol-v4x0.test.js

```diff
@@ -17,25 +17,26 @@
  * limitations under the License.
  */
 
-import BoltProtocolV4x0 from '../../../bolt-connection/lib/bolt/bolt-protocol-v4x0'
-import RequestMessage from '../../../bolt-connection/lib/bolt/request-message'
+import BoltProtocolV4x0 from '../../src/bolt/bolt-protocol-v4x0'
+import RequestMessage from '../../src/bolt/request-message'
 import utils from '../test-utils'
-import { WRITE } from '../../../src/driver'
 import {
   ProcedureRouteObserver,
   ResultStreamObserver
-} from '../../../bolt-connection/lib/bolt/stream-observers'
+} from '../../src/bolt/stream-observers'
 
 import { internal } from 'neo4j-driver-core'
 
+const WRITE = 'WRITE'
```

Comment:
Not sure why this variable is needed, the string seems to be inlined in other changes

Reply:
Actually I prefer this `const` solution, it's cleaner and easy to change if the value change.